src/c++/perf_analyzer/genai-pa/genai_pa/llm_metrics.py

```diff
@@ -121,6 +123,69 @@ def __repr__(self):
         attr_strs = ",".join([f"{k}={v}" for k, v in self.__dict__.items()])
         return f"Statistics({attr_strs})"
 
+    def _is_time_field(self, field: str):
+        time_fields = [
+            "inter_token_latency",
+            "time_to_first_token",
+            "end_to_end_latency",
+        ]
+        return field in time_fields
+
+    def _use_exact_length(self, value: str):
+        exact_value_length = 17
+        formatted_value = f"{value:.{exact_value_length}f}"
+        if len(formatted_value) < exact_value_length:
+            return formatted_value.ljust(exact_value_length)
+        return formatted_value[:exact_value_length]
+
+    def pretty_print(self):
+        table = Table(title="PA LLM Metrics")
+
+        table.add_column("Statistic", justify="right", style="cyan", no_wrap=True)
+        stats = ["avg", "min", "max", "p99", "p95", "p90", "p75", "p50", "p25"]
```

Comment:
Table looks nice!

Question on `min` field I observed in the output - it is drastically smaller than average. Is this coming from time between response N-1 and response N (final empty)?

Reply:
Good question.
I think this is a question for other team members.
I am gonna move this offline.

Reply:
Sounds good, assuming you'll follow up on it. Resolving.